neptune/internal/backends/hosted_neptune_backend.py

```diff
@@ -682,6 +682,17 @@ def extract_experiment_output(self, experiment, data):
                 raise StorageLimitReached()
             raise
 
+    @with_api_exceptions_handler
+    def rm_data(self, experiment, paths):
+        try:
+            return [
```

Comment:
Loop should be in Experiment instead of Client since our api endpoint does not accept list of paths.